Infrastructure/SetEnvironmentSecretsTemplate.sh

```diff
@@ -27,4 +27,7 @@ cf set-env "sea-beta-${PAAS_ENV_SHORTNAME}" Bre__Password "<REAL_VALUE_HERE>"
 # GovUK Notify Credentials
 cf set-env "sea-beta-${PAAS_ENV_SHORTNAME}" GovUkNotify__ApiKey "<REAL_VALUE_HERE>"
 
+# Google Analytics Credidentials
+cf set-env "sea-beta-${PAAS_ENV_SHORTNAME}" GoogleAnalytics__ApiSecret "<REAL_VALUE_HERE>"
+
```

Comment:
We need to run this on the environments before this ticket is merged

Reply:
I'll let you merge the ticket once the environment is set up